packages/analytics-types/src/config.ts

```diff
@@ -47,6 +47,8 @@ export interface BrowserConfig extends Config {
   trackingOptions: TrackingOptions;
 }
 
+export type NodeConfig = Config;
```

Comment:
do we need this then?

Reply:
actually can we base the config off of https://github.com/amplitude/Amplitude-Python/blob/main/src/amplitude/config.py#L15 and https://github.com/amplitude/Amplitude-Node/blob/main/packages/types/src/options.ts#L10. some config may not be needed and should just be moved to browser config